src/FPIndia/View.hs

```diff
@@ -53,12 +53,15 @@ renderBody rp model r = do
         renderMarkdown model "resources.md"
     renderFooter
 
-renderNavbar :: Prism' FilePath Route -> H.Html
-renderNavbar rp =
+renderNavbar :: Prism' FilePath Route -> HtmlRoute -> H.Html
+renderNavbar rp r =
   H.nav ! A.class_ "w-full h-1/4 text-xl font-bold flex space-x-4  mb-4" $ do
-    forM_ universe $ \r -> renderURL (H.toHtml $ routeTitle r) (routeHref rp r)
+    forM_ universe $ \x -> if x == r then renderURL (H.toHtml $ routeTitle x) (routeHref rp x) True else renderURL (H.toHtml $ routeTitle x) (routeHref rp x) False
```

Comment:
`x` is too generic. I'd keep it as `r` and rename the function's argument as `currentRoute`.

Reply:
Or you can rename the nav route argument as `navRoute`.